### PR TITLE
Count public & private requests

### PR DESCRIPTION
**Pull request by gwicke**
Keep a simplistic count of requests from public & private networks.

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -192,6 +192,14 @@ function handleRequest (opts, req, resp) {
     };
     reqOpts.log = reqOpts.logger.log.bind(reqOpts.logger);
 
+    // Simplistic count of requests from public & private networks
+    var remoteAddr = req.socket.remoteAddress;
+    if (/^(?:10|127)\./.test(remoteAddr)) {
```
Comment (Pchelolo) [inline lib/server.js]: What about 172.16.x.x - 172.31.x.x and 192.168.x.x?

Comment (d00rman) [inline lib/server.js]: `10.x.x.x` as well

Comment (d00rman) [inline lib/server.js]: Ups, it's there already. :P

File: lib/server.js [hunk 2]
```diff
@@ -192,6 +192,14 @@ function handleRequest (opts, req, resp) {
     };
     reqOpts.log = reqOpts.logger.log.bind(reqOpts.logger);
 
+    // Simplistic count of requests from public & private networks
+    var remoteAddr = req.socket.remoteAddress;
```
Comment (d00rman) [inline lib/server.js]: `remoteAddress` will report the connecting *client* which will be the LVS (or Varnish). In that respect, you are right to count `127.x.x.x`, but `10.x.x.x` will cover all of the remaining requests, which means we'll have 100% of private requests.

Varnish sets the `X-Forwarded-For` header, so we should use that one, and, if it's missing, use the socket remote address.

File: lib/server.js (hunk 1 above)
Comment (gwicke) [inline lib/server.js]: We don't use the others ;) But sure, lets add them too.

File: lib/server.js (hunk 2 above)
Comment (gwicke) [inline lib/server.js]: Hmm, yeah. Good point ;)

**Comment by gwicke** [comment]
Added support for XFF as well.

**Pull request merged**